pom.xml

```diff
@@ -291,7 +291,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.7.5.201505241946</version>
+                <version>0.8.4</version>
```

Comment:
ℹ️ FYI: JaCoCo 0.8.5 was released on Oct 11, 2019.

Reply:
Dependency has been upgraded to version 0.8.7